Parsers/ASimRegistryEvent/Parsers/imRegistry.yaml

```diff
@@ -1,7 +1,7 @@
 Parser:
   Title: Registry Event ASIM Parser
-  Version: '0.1'
-  LastUpdated: June 23, 2021
+  Version: '0.1.1'
```

Comment:
You actually did not change this one.

Reply:
Sorted this one. Was related to the other PR